### DISCUSSION TITLE
Configuration For Login Challenge Expired Time

### DISCUSSION THREAD

**Discussion by simonwjy** [ory/hydra#2475] [Q&A] [answered]
Hi Experts, 

I'm wondering is there any way that we can set the expired time for login challenge? what is the default time for the challenge to be expired?

Thank you!

**Comment by Benehiko** [comment, marked as answer]
Hi @SimonWangJY 

Thanks for reaching out :smile: 

Do you mean the login/consent request expiration? The default for that is 1 hour
https://www.ory.sh/hydra/docs/guides/token-expiration#login-and-consent-request-expiration

**Reply by vinckr** [reply]
Hey @SimonWangJY ,
Just checking in if your question was answered or if there was still something unclear.
I will mark this as answered for now, but feel free to come back to the discussion any time.

Thanks a ton! 🐝
